packages/babel-types/src/builders/builder.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import loClone from "lodash/clone";
+import loClone from "lodash.clone";
```

Comment:
Can we use `Object.assign()` or spread operator (`const clone = { ...obj }` or `const clone = [...arr]`) since this is a shallow clone? That would allow us to remove this dep.

Reply:
Although `uniq` and `values` look fairly safe to replace (see https://github.com/babel/babel/pull/11790), `clone` seems a little more subtle (for example, due to the fact that `lodash/clone` handles different types automatically, and also handles basic types).

I haven't yet applied any changes there, although with a bit more scrutiny and care it's probably possible to remove the usage of `lodash/clone` here, yep.

Reply:
(removal of `lodash/clone` now handled via https://github.com/babel/babel/pull/11811)